submissions/zhenyakornilov/js-dom/js/index.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+import museums from "./data.js";
+import switchTheme from "./utils.js";
+
+function main() {
+  switchTheme();
+
+  const listMenu = document.querySelector(".menu-items");
+  const contentBody = document.querySelector(".content-body");
+  renderButtons(museums, listMenu);
+  toggleSelectedButton(museums, contentBody);
+  openMenu(listMenu);
+}
+
+document.addEventListener("DOMContentLoaded", main);
+
+function renderButtons(arr, menu) {
```

Comment:
arr sounds too generic. We can name it `listOfCards` for example or what you prefer more